backend/app/agents/devrel/tools/search_tool.py

```
@@ -0,0 +1,41 @@
+            for result in response.get("results", []):
+                results.append({
+                    "title": result.get("title", ""),
+                    "content": result.get("content", ""),
+                    "url": result.get("url", ""),
+                    "score": result.get("score", 0)
+                })
+
+            logger.info(f"Search for '{query}' returned {len(results)} results")
+            return results
+
+        except Exception as e:
```

Pylint (3.3.7)

[warning] Catching too general exception Exception

(W0718)